src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/PartnershipControllerTest.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.controller;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.mock.web.MockHttpServletRequest;
+import uk.gov.companieshouse.limitedpartnershipsapi.model.dto.LimitedPartnershipSubmissionCreatedResponseDto;
+
+import java.util.Objects;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+class PartnershipControllerTest {
+
+    @InjectMocks
+    private PartnershipController partnershipController;
+
+    @Mock
+    private MockHttpServletRequest mockHttpServletRequest;
+
+    @Test
+    void testCreatePartnership() {
```

Comment:
Will there be a test for the failure scenario?

Reply:
yes, once the story is finished it should tests to cover all scenarios.